tokens/ETH.json

```diff
@@ -263,6 +263,14 @@
     "name": "EURA (previously agEUR)",
     "logoURI": "https://raw.githubusercontent.com/AngleProtocol/angle-token-list/main/src/assets/tokens/EURA.svg"
   },
+  {
+    "address": "0x004626A008B1aCdC4c74ab51644093b155e59A23",
+    "chainId": 1,
+    "logoURI": "https://raw.githubusercontent.com/AngleProtocol/angle-token-list/main/src/assets/tokens/stEUR.svg",
+    "decimals": 18,
+    "name": "Staked EURA",
+    "symbol": "stEUR"
+  },
```

Comment:
Why do we add stEUR and stUSD in this PR?

Reply:
They are not added but just the order is replaced.

Reply:
I assume my local setup was different for them cause I added them a few weeks ago.